content/news/playtest-20180101.md

```diff
@@ -0,0 +1,64 @@
+
+ * The full Harkonnen campaign is now available in D2K
+ * Several campaign missions in Red Alert and Tiberian Dawn have been refined and improved
+ * The AI now properly accounts for visibility when selecting superweapon and support power targets
+ * Several other changes to the AI should improve their behaviour and performance
+
+<div style="text-align:center" markdown="1">
+![AI Airstrike](/images/news/20180101-airstrike.png)
+
+Improvements to the AI support power targeting mean that this will be a rarer sight.
+ </div>
+
+Other notable changes include:
+
+* New lobby options to disable bounty, build radius, and re-packing Construction Yards
+* Units will no longer target units revealed by the Red Alert GPS Satellite
+* A collection of community-driven balance changes for Red Alert
+* Fixes for the turret glitches experienced by the RA V2 and TD Rocket Launchers
+* Fixes for many other graphical glitches and polish issues
+* Fixes for a number of platform integration issues on Windows and macOS
+* Further improvements to memory usage and performance
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180101-mapoptions.png" width="600" alt="Expanded game options">
+
+The Map Options tab includes several new settings, and can be customized by scripted maps.
+</div>
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180101-winmacos.png" width="600" alt="Windows and macOS improvements">
+
+Platform integration fixes include "Pin to taskbar" support on Windows and 64 bit support for macOS.
+</div>
+
+
+As always, we suggest reading the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog) for more information and to view the full list of changes.
+We make these playtests available to get feedback from the community about the future direction of OpenRA, and we value your opinions.
+Let us know what you think and please report any bugs you discover via [our forum](http://www.sleipnirstuff.com/forum/viewforum.php?f=80), our [GitHub issue tracker](https://github.com/OpenRA/OpenRA/issues), or in the comments below.
+
+OpenRA playtest builds do not overwrite the release version if you are on Windows or macOS, and you can switch between the two when joining an online server. So head over to our [download page](/download/) and give it a try!
+
+The last development cycle has brought some significant improvements for modders, but we recommend that projects currently based on the stable release to hold off upgrading for now. Experienced OpenRA modders well know the difficulties of updating your code for a new OpenRA release, and this playtest includes more changes than most. We are planning major improvements to our semi-automatic mod-upgrader to be included in a future playtest, and we will update the [OpenRA Mod SDK](https://github.com/OpenRA/OpenRAModSDK) when these changes are ready. Stay tuned for more info!
```

Comment:
In that first sentence I'd remove the 'to' before and add 'from' after the 'hold off'.